jupyter_forward/core.py

```diff
@@ -204,7 +204,7 @@ def _launch_jupyter(self):
         command = r'jupyter lab --no-browser --ip=\$(hostname -f)'
         if self.notebook_dir:
             command = f'{command} --notebook-dir={self.notebook_dir}'
-        command = f'{command} >& {self.log_file}'
+        command = f'{command} > {self.log_file} 2>&1'
```

Comment:
I'm concerned that this change is not compatible with `tcsh` and is actually reintroducing a bug fixed in [#67](https://github.com/ncar-xdev/jupyter-forward/pull/67/files#diff-441d52a1bbe59587631e24cd2a5593520f482d0d78737cbf0f75381dab82ecb7R184). @tobias-haenel what sort of error do you see if you add the `-l` argument but keep the `>&` notation?

@andersy005 we should try to figure out how to create a test suite for `jupyter-forward` to prevent regressions like this... I suspect it can't be fully automated, but we definitely need to make sure all PRs are compatible with both `bash` and `tcsh` before accepting them